### PR TITLE
Check for survey name on field close messages

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
We were missing `surveyName` on outbound field `CLOSE` messages, tests now check for it.

# What has changed
* Check survey name on field close messages
* Update language to reflect action instruction changes

# How to test?
Build and run with linked fieldwork adapter branch

# Links
https://trello.com/c/nwGrBR4Q/639-tweak-outbound-fwmt-close-msg-to-match-interface
https://github.com/ONSdigital/census-rm-fieldwork-adapter/pull/32

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Tweak outbound FWMT close msg to match interface](https://trello.com/c/nwGrBR4Q/639-tweak-outbound-fwmt-close-msg-to-match-interface)

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: acceptance_tests/features/steps/receipt.py [hunk 1]
```diff
@@ -54,15 +54,15 @@ def uac_updated_msg_emitted(context):
     test_helper.assertFalse(emitted_uac['active'])
 
 
-@step('an ActionCancelled event is sent to field work management with addressType "{address_type}"')
-@step('an ActionCancelled Invalid Address event is sent to field work management with addressType "{address_type}"')
+@step('a CLOSE action instruction is sent to field work management with addressType "{address_type}"')
 def action_cancelled_event_sent_to_fwm(context, address_type):
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/receipt.py]: You could change the method name to action closed instead

Comment (AdamHawtin) [reply acceptance_tests/features/steps/receipt.py]: 👍 Too easy to forget the method names, good spot

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍 just left a small comment but I would stop it from being merged

**Pull request merged**